testsuite/OMSimulator/deleteConnector.lua

```diff
@@ -0,0 +1,241 @@
+-- status: correct
+-- linux: yes
+-- mingw: yes
+-- win: yes
+-- mac: yes
+
+oms_setCommandLineOption("--suppressPath=true")
+oms_setTempDirectory("./deleteConnector_lua/")
+
+oms_newModel("deleteConnector")
+
+oms_addSystem("deleteConnector.Root", oms_system_wc)
+oms_addSystem("deleteConnector.Root.System1", oms_system_sc)
+oms_addConnector("deleteConnector.Root.System1.C1", oms_causality_input, oms_signal_type_real)
+oms_addConnector("deleteConnector.Root.System1.C2", oms_causality_input, oms_signal_type_real)
+
+oms_addSystem("deleteConnector.Root.System2", oms_system_sc)
+oms_addConnector("deleteConnector.Root.System2.C3", oms_causality_output, oms_signal_type_real)
+oms_addConnector("deleteConnector.Root.System2.C4", oms_causality_output, oms_signal_type_real)
+oms_setReal("deleteConnector.Root.System2.C3", 20.0)
+oms_setReal("deleteConnector.Root.System2.C4", 30.0)
+
+oms_addSubModel("deleteConnector.Root.System1.Gain", "../resources/Modelica.Blocks.Math.Gain.fmu")
+oms_setReal("deleteConnector.Root.System1.Gain.k", 30.0)
+
+oms_addConnection("deleteConnector.Root.System2.C3", "deleteConnector.Root.System1.C1")
+oms_addConnection("deleteConnector.Root.System2.C4", "deleteConnector.Root.System1.C2")
+
+src = oms_list("deleteConnector")
+print(src)
+
+-- delete connector "deleteConnector.Root.System2.C3" and associated connection
+oms_delete("deleteConnector.Root.System2.C3")
+
+-- delete components connector and its startValue
+oms_delete("deleteConnector.Root.System1.Gain.k")
```

Comment:
Are we actually allowed to delete this, since `k` is part of the FMU?

Reply:
@adeas31 , yes we are allowed to do this,  as different tools does not export all the variables related to FMUs, some tools export only inputs and outputs